### DISCUSSION TITLE
About using stack model in Pycaret

### DISCUSSION THREAD

**Discussion by Linyi-Chen** [pycaret/pycaret#3984] [General]
I have a problem using stack_models fuction. When I try to run this, It will say that ValueError: cannot set WRITEABLE flag to True of this array.

![a20dfc669e586afc2c260c46fb0e964](https://github.com/pycaret/pycaret/assets/156176256/a89486f7-0f91-43f5-8cc6-29a61cca08d3)

**Comment by jadencodes** [comment]
I figured out a workaround for this. There is a bug in 1.4.2 mentioned here: https://github.com/scikit-learn/scikit-learn/issues/28781

The workaround is to use scikit-learn 1.4.0. It may be possible to use 1.4.1, but it is no longer available (only 1.4.1.post1 which still has the regression).

e.g. 
```
pip install --upgrade scikit-learn==1.4.0
```
